modules/roma-number/include/romanumber_converter.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 Pinaev Danil
+
+#ifndef MODULES_ROMA_NUMBER_INCLUDE_ROMANUMBER_CONVERTER_H_
+#define MODULES_ROMA_NUMBER_INCLUDE_ROMANUMBER_CONVERTER_H_
+
+#include <string>
+#include "include/romanumber.h"
+
+class RomaNumberConverter {
+ public:
+    RomaNumberConverter();
```

Comment:
Может все же неправильно? У всех работает
```suggestion
    RomaNumberConverter() = default;
```